aws-codeartifact-repository/aws-codeartifact-repository.json

```diff
@@ -0,0 +1,84 @@
+{
```

Comment:
please add the appropriate `required` properties.

Reply:
will do